frontend/src/prompted_readings/analysis_view.js

```diff
@@ -1,4 +1,34 @@
 import React from "react";
+import PropTypes from "prop-types";
+
+class SentimentScores extends React.Component {
```

Comment:
Please add a render test for this component -- see examples in the test file in frontend/src/prompted_readings